packages/common/amplify_db_common/android/build.gradle

```diff
@@ -41,13 +41,13 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 19
```

Comment:
Just double checking we're okay to bump this too

Reply:
Yeah it’s 24 everywhere else